### ISSUE TITLE
Tooltip sizing issue on static portions of page

### ISSUE THREAD

**Issue by msosa** [ng-bootstrap/ng-bootstrap#3046]
### Bug description:
Tooltips on a static portion of a page gets jumbled when you scroll the page. Maybe the scrolled page messes up the calculations of the tooltip?

Also note that this only happens when `container=body`. But in my use case I cannot remove that.

### Link to minimally-working StackBlitz that reproduces the issue: 
https://stackblitz.com/edit/angular-nr8fba-ze8qba
(make sure to scroll the page horizontally)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 7.2.6

ng-bootstrap: 4.0.1

Bootstrap: 4.0.0

**Comment by maxokorokov** [comment]
@msosa you're scrolling the body, but the tooltip is attached to the body and not repositioned on scroll. This is true to all popup-widgets currently. 

I think the options we have now are:
- close immediately or reposition on any parent scroll (#3018, #3043)
- allowing to provide alternative container (ex. #1884)

**Comment by msosa** [comment]
@maxokorokov are you saying it won't ever be repositioned after its been scrolled, or only during the actual scroll?

Would closing it immediately fix it? Once it reopens the tooltip body will still be distorted.

But yeah I think an alternative container would probably work.

**Comment by msosa** [comment]
FWIW js bootstrap seems to handle the creating of the scrolled tooltip ok

https://codepen.io/msosa/full/WmGwmM

**Comment by msosa** [comment]
hmm @maxokorokov I did notice that bootstrap js something other than `top`/`left` to position their tooltips. So I decided to try it(only in browser console) and it seems to work. Although the math is slightly off for my one testing scenario

![image](https://user-images.githubusercontent.com/1843410/54007033-eacabe00-412d-11e9-8a4b-18d9ea13624a.png)


in this case the left was off by 50px. Maybe its possible to use `translate3d`?

**Comment by msosa** [comment]
Heh after looking at the code it looks like my main issue has just recently been fixed in #2972 where transform was added.

But what you mention about the tooltip staying not moving with the scroll is still happening. 

Also while I was running the demo locally I noticed a small error in https://github.com/ng-bootstrap/ng-bootstrap/blob/master/DEVELOPER.md#yarn-demo.  I think once ng-bootstrap switched over to angular-cli the port number changed to `4200` from `9090`
